src/features/Layout/Layout.tsx

```diff
@@ -124,13 +146,33 @@ const CenteredBannerText = styled.div`
   }
 `;
 
+const DemoBanner = styled.div`
+  position: absolute;
+  background: ${COLORS.WARNING_ORANGE};
+  & p {
+    font-weight: bold;
```

Comment:
Why is this styling not defined in the banner itself?

Reply:
We don't use the `Banner` component here because it's not really a banner when compared to the other types of banners we use. It's more of an text box attached to the wrapper around the whole app view. Not really sure what to call it other than "banner"

Reply:
Can it be moved to a component then?